src/dex/nerve/config.ts

```diff
@@ -0,0 +1,467 @@
+import { SwapSide } from 'paraswap-core';
+import type { AbiItem } from 'web3-utils';
+import { AdapterMappings, DexParams } from './types';
+import { DexConfigMap } from '../../types';
+import { Network } from '../../constants';
+const nervePoolABI = require('../../abi/nerve/nerve-pool.json');
+const axialPoolABI = require('../../abi/nerve/axial-pool.json');
+const ironV2PoolABI = require('../../abi/nerve/iron-v2-pool.json');
```

Comment:
why not use import <name> from '<abi-location>'?

Reply:
I just wanted to get autocompletion for functions on Contracts, but it didn't work. Replaced with ordinary `import <name> from '<abi-location>'`